Vic3ToHoI4Tests.vcxproj

```diff
@@ -45,13 +45,13 @@
   <PropertyGroup Label="UserMacros" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
     <LinkIncremental>true</LinkIncremental>
-    <OutDir>build/test/$(Configuration)/output/</OutDir>
-    <IntDir>build/test/$(Configuration)/intermediate/</IntDir>
+    <OutDir>$(SolutionDir)build/test/$(Configuration)/</OutDir>
```

Comment:
You really should drop debug altogether unless you intend to use it.

Reply:
Opened an issue about that